.lune/fmt.luau

```diff
@@ -0,0 +1,7 @@
+--> Run stylua to check for formatting errors
+
+local process = require("@lune/process")
+
+local CommandBuilder = require("./lib/exec")
+
+process.exit(CommandBuilder.new("stylua"):withArg("."):withArgs(process.args):withStdioStrategy("forward"):exec().code)

```

Comment:
Stylua isn't in the manifest. Probably Rokit managed here too.

Reply:
Addressed in https://github.com/pesde-pkg/scripts/pull/3/commits/5e0905673164aeedb025027d7fc484a7438111b6.